src/main/java/com/eternalcode/combat/config/ConfigService.java

```diff
@@ -0,0 +1,43 @@
+package com.eternalcode.combat.config;
+
+import eu.okaeri.configs.ConfigManager;
+import eu.okaeri.configs.OkaeriConfig;
+import eu.okaeri.configs.serdes.commons.SerdesCommons;
+import eu.okaeri.configs.yaml.bukkit.YamlBukkitConfigurer;
+
+import java.io.File;
+import java.util.HashSet;
+import java.util.Set;
+
+public class ConfigService {
+
+    private final ConfigBackupService backupService;
+
+    private final Set<OkaeriConfig> configs = new HashSet<>();
+
+    public ConfigService(ConfigBackupService backupService) {
+        this.backupService = backupService;
+    }
+
+    public <T extends OkaeriConfig> T create(Class<T> config, File file) {
+        this.backupService.createBackup();
+
+        T configFile = ConfigManager.create(config, (it) -> {
```

Comment:
`it` zmień na to czym to jest (setting config?)
możesz wszystko zapisać w chain

Reply:
To skrót od "initializer".

Reply:
No to tak nazywaj zmienne. Nie stosujemy w większości przypadków skrótów